core/services/ocr2/plugins/ocr2keeper/evm21/feed_lookup.go

```diff
@@ -96,52 +98,71 @@ func (r *EvmRegistry) feedLookup(ctx context.Context, upkeepResults []EVMAutomat
 			continue
 		}
 
-		feedLookup, err := r.decodeFeedLookup(upkeepResults[i].PerformData)
+		r.lggr.Infof("[FeedLookup] upkeep %s block %d decodeFeedLookup performData=%s", upkeepId, block, hexutil.Encode(upkeepResults[i].PerformData))
```

Comment:
nit. will this always be 'block'? As i understand it can be timestamp also based on user revert data?

Reply:
this block is from `block := upkeepResults[i].Block` so this is the block which OCR3 has agreed upon.
the block/time from the revert data is denoted as `time` in `FeedLookup` struct

Reply:
ah might be better to rename to `checkBlock` to clarify